### PR TITLE
[libev] change package name to unofficial-libev

### PR DESCRIPTION
**Pull request by ybogo**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [x] The "supports" clause reflects platforms that may be fixed by this new version
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

This PR relates to review comment https://github.com/microsoft/vcpkg/pull/32278#issuecomment-1624182345

### CODE REVIEW

File: ports/libev/portfile.cmake [hunk 1]
```diff
@@ -23,5 +23,5 @@ file(
     DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}"
     RENAME copyright
 )
-file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/libevConfig.cmake"
-     DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/unofficial-libev-config.cmake"
```
Comment (LilyWangLL) [inline ports/libev/portfile.cmake:26]: Please also change the following file name.
```
[VCPKG_ROOT_PATH]/ports/libev/libevConfig.cmake
```

**Review by LilyWangLL — CHANGES_REQUESTED** [review]

**Comment by LilyWangLL** [comment]
Thanks for your PR. Please update the port-version in `vcpkg.json`. See more from: https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md#update-the-port-version-field-in-the-manifest-file-of-any-modified-ports

**Comment by LilyWangLL** [comment]
Note: I will be converting your PR to draft status. When you respond, please revert to "ready for review". That way, I can be aware that you've responded since you can't modify the tags.

File: ports/libev/portfile.cmake (hunk 1 above)
Comment (ybogo) [reply ports/libev/portfile.cmake:26]: Can you recheck? It is already renamed :)

**Comment by ybogo** [comment]
@LilyWangLL, thanks for your comments! I didn't know about duplicate PR. Anyway I made changes you requested. So let me know if this PR should be closed.

**Comment by JonLiu1993** [comment]
@ybogo, thanks for the PR, please keep your PR, I will close mine.

**Review by LilyWangLL — DISMISSED** [review]

**Review by BillyONeal — APPROVED** [review]

**Pull request merged**